docs/extensions/guides/ipc.md

```diff
@@ -40,38 +40,46 @@ The general terminology is as follows:
 
 To register either a handler or a listener, you should do something like the following:
 
+File1:
 ```typescript
 import { LensMainExtension, Interface, Types } from "@k8slens/extensions";
 
 export class ExampleExtensionMain extends LensMainExtension {
   onActivate() {
-    this.disposers.push(
-      this.listenIpc({
-        channel: "initialize",
-        listener: this.initializeListener,
-        verifier: this.initializeVerifier,
-      })
-    );
+    this.listenIpc({
+      channel: "initialize",
+      listener: this.initializeListener,
+      verifier: this.initializeVerifier,
+    });
   }
 
-  initializeListener = (event: Types.IpcMainEvent, uid: string) => {
-    console.log(`starting to initialize: ${uid}`);
+  initializeListener = (event: Types.IpcMainEvent, id: string) => {
+    console.log(`starting to initialize: ${id}`);
   };
 
-  initializeVerifier = (args: unknown[]): args is [uid: string] => {
+  initializeVerifier = (args: unknown[]): args is [id: string] => {
     return args.length === 1 && typeof args[0] === "string";
   }
 }
 ```
 
-If you want to register a "handler" you would call `this.handleIpc(...)` instead.
+File2:
```

Comment:
Can we call this `renderer.ts` instead of "File1" so it's clear this is code you could put on the `renderer` thread?